extensions/resteasy-reactive/rest-jackson/deployment/src/test/java/io/quarkus/resteasy/reactive/jackson/deployment/test/SimpleJsonResource.java

```diff
@@ -52,6 +52,7 @@ public Response handleParseException(WebApplicationException e) {
 
     @GET
     @Path("/person")
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
Why were these `@Produces` added? They are not necessary in Quarkus and if the test now fails without them, that is a genuine issue

Reply:
There isn't any test failure without that annotation, but as you can see [here](https://github.com/quarkusio/quarkus/pull/41063/files#diff-809ac962192212e1560ac994ce31cdecdb93e1dd6d0cee71011fccbb063e728dR387), I'm adding the type returned among the ones for which I generate the serializers only if that annotation is present. Should I ignore it and generate the serializer regardless?

Reply:
That approach is going to be a problem because in Quarkus REST we use JSON by default. See `JsonDefaultProducersHandler`

Reply:
No problem, I will generate the serializer regardless if the `@Produces` is present or not and also remove it from tests.

Reply:
👍🏼